core/java/com/google/instrumentation/common/Duration.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.common;
+
+/**
+ * Represents a signed, fixed-length span of time represented as a count of seconds and fractions
```

Comment:
this looks very familiar. are there any other hrefs you can cite for this exact practice used elsewhere?